src/components/TodoCard/TodoCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import IsFinished from '../IsFinished/IsFinished';
+import IsImportant from '../IsImportant/IsImportant';
+
+import './TodoCard.css'
+
+function TodoCard({ todo, ...props }) {
```

Comment:
Here you define rest props but don't use it anywhere. Just remove it.
If you don't use something, it's better to remove it.
Avoid creating unused variables, it makes you code more complex and less readable.